### ISSUE TITLE
Short links are unreliable

### ISSUE THREAD

**Issue by RidiculousHat** [oliverfei/HearthstoneDeckViewer#3]
The short link redirect service intermittently provides 504 errors and leaves the decklists inaccessible. The issues are unpredictable, leaving the short link service difficult to use and unreliable - it may be worth removing until this is resolved.

**Comment by oliverfei** [comment]
Hey, thanks for the heads up. Checked it just now, and it seems to be working alright. That said, there's certainly no harm in adding a simple health check for the link shortener service.

**Comment by RidiculousHat** [comment]
Yeah, the main issue is that it generally works - but sometimes it just completely breaks temporarily with no warning and then fixes itself over time. A health check sounds like a good idea. :)
